test/createFunction/createFunction.Script.v2.test.ts

```diff
@@ -161,36 +160,34 @@ function addSuite(tester: FunctionTesterBase): void {
             );
         });
 
-        // For now - durable is only supported in these languaeges
-        if (tester.language === ProjectLanguage.TypeScript || tester.language === ProjectLanguage.JavaScript) {
-            const durableActivity: string = 'Durable Functions activity';
-            test(durableActivity, async () => {
-                await tester.testCreateFunction(
-                    durableActivity
-                );
-            });
+        function fixDurableLabel(label: string): string {
+            if (tester.language === ProjectLanguage.PowerShell || tester.language === ProjectLanguage.Python) {
```

Comment:
So I thought before only TypeScript and JavaScript were supported.  I see we have 2 new languages in preview, but it seems like all other languages support Durable Functions now?

Reply:
This file is createFunction.**_Script_**.v2.test.ts. Aka it only covers TS, JS, Python, and PowerShell anyways. The others (Java, C#) are handled in a separate test file